### DISCUSSION TITLE
Slow resore process with kraft

### DISCUSSION THREAD

**Discussion by KubovD** [strimzi/strimzi-kafka-operator#10366] [Q&A]
Hello.
After we switched kafka cluster from zookeper to kraft we get issue. 
We made dump of the disk and transfer kafka with same configuration of nodes to another k8s, but after start entity operator "read" topics messages and offsets very slow, what never happens with zookeper. Is it possible to avoid this problem?

**Comment by scholzj** [comment]
The entity operator does not read any messages - not with KRaft, where you have to use the Unidirectional Topic Operator.
